src/templates/en/2019/contributors.html

```diff
@@ -107,20 +171,21 @@ <h1>
     <div class="contributor-grid">
     {% for id, contributor in config.contributors.items() %}
       <div class="contributor {{ ' '.join(contributor.teams) }}">
-        <img class="contributor-avatar" src="{{ contributor.avatar_url }}" height="100" width="100" alt="{{ contributor.name }} avatar">
+        <img class="contributor-avatar" src="{% if contributor.avatar %}{{contributor.avatar}}&s=200{% else %}data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mNkYAAAAAYAAjCB0C8AAAAASUVORK5CYII=
+{% endif %}" height="100" width="100" alt="{{ contributor.name }} avatar" loading="lazy">
         <div class="contributor-name">{{ contributor.name }}</div>
         {% if contributor.twitter or contributor.github %}
         <div class="contributor-social">
```

Comment:
Some contributors have no social links. If possible this space would be empty so the team names are aligned across the row of contributors.

![image](https://user-images.githubusercontent.com/1120896/67814053-7c01f280-fa79-11e9-8d69-4905e024319f.png)